app/components/common/eventCard/index.test.tsx

```diff
@@ -0,0 +1,31 @@
+import { render, screen } from '@testing-library/react';
+import EventCard from './index';
+
+describe('EventCard', () => {
+  const eventDetails = {
+    date: 15,
+    month: 'March',
+    fromTime: '9:00 AM',
+    toTime: '11:00 AM',
+  };
+  const eventTitle = 'Test Event';
+  const participantCount = 3;
```

Comment:
please use the object structure same as it will be coming from backend. Otherwise this will be obselete in real data.

Reply:
Done, thanks for pointing out @iRohitGaur !